src/test/java/com/thealgorithms/maths/ConvolutionTest.java

```diff
@@ -0,0 +1,74 @@
+package com.thealgorithms.maths;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ConvolutionTest {
+
+    @Test
```

Comment:
Will be better to improve these tests using `@ParameterizedTest` and using additional record or class with data for TestCase. There are a lot of examples in repository

Reply:
Done!